src/types.ts

```diff
@@ -112,4 +169,21 @@ export interface ICourierClient {
   getProfile: (
     params: ICourierProfileGetParameters
   ) => Promise<ICourierProfileGetResponse>;
+  getBrands: (
+    params?: {
+      cursor: string
+    }
+  ) => Promise<ICourierBrandGetAllResponse>;
+  getBrand: (
+    brandId: string
+  ) => Promise<ICourierBrand>;
+  createBrand: (
+    params: ICourierBrandPostParameters
+  ) => Promise<ICourierBrand>;
+  replaceBrand: (
+    params: ICourierBrandPutParameters
+  ) => Promise<ICourierBrand>;
+  deleteBrand: (
+    brandId: string
+  ) => Promise<any>;
```

Comment:
I would avoid `any` in general. Is `void` a good replacement here?

Reply:
Perhaps `void` is what is needed.